config-secrets/templates/redis-config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
What is the purpose of this file? There is already `redis-configmap.yaml` and is used in the chapter.

Reply:
Don't recall. This maybe during one of those workshop fixes. I will delete on my end as it may be fixed

Reply:
Don't see ~$slides.pptx anymore.